### ISSUE TITLE
Если запускать gemini через gulp-shell, то html отчет показывает всё по нулям

### ISSUE THREAD

**Issue by varya** [bem/bem-forum-content-ru#523]
Я запускаю Gemini внутри gulp вот так:

```
.pipe(shell([
'./node_modules/gemini/bin/gemini test --reporter html --reporter flat ./tests/gemini-test.js'
]),

shell — это плагин gulp-shell, но это не важно, потому что с run и exec то же самое.

Тесты проходят, в консоль есть вывод:

```
Total: 192 Passed: 136 Failed: 56 Skipped: 0
```
 
А вот для html-репорта показывает вот это: https://img-fotki.yandex.ru/get/3812/14441195.34/0_93389_de3023ad_orig
При этом все нужные картинки сняты и диффы правильные.

Как мне это починить?

/cc @sevinf @arikon

**Comment by SevInf** [comment]
Очень интересный баг. Я правильно понял, что в `gemini-report/images` есть все что нужно и проблема в только в `gemini-report/index.html`?

**Comment by varya** [comment]
@sevInf да, все картинки снимаются нормально. Сам файл `index.html` тоже пишется на диск. Но там по нулям.

**Comment by varya** [comment]
Ой, забыла сказать, что если ту же самую команду запускать не через Gulp а вручную в консоли, то содержание `index.html` такое как нужно.

**Comment by varya** [comment]
Так что? Это баг в Gemini и на него будет заведен таск?
Или можно как-то по-другому Gemini вызывать через Gulp, чтобы этого бага избежать?

**Comment by SevInf** [comment]
Завел [баг](https://github.com/gemini-testing/gemini/issues/196).
Пока могу предложить попробовать заиспользовать [API](https://github.com/gemini-testing/gemini/blob/master/doc/programmatic-api.md). Я не знаю правда, насколько сложно его под Gulp адаптировать будет.
